cob_frame_tracker/src/cob_frame_tracker.cpp

```diff
@@ -319,7 +339,9 @@ void CobFrameTracker::publishTwist(ros::Duration period, bool do_publish)
 
     if (do_publish)
     {
+        ROS_INFO_STREAM("[publishTwist] Period: " << period.toSec());
```

Comment:
we should remove this again as it is producing to much terminal output...by using it locally we already proofed that the period computation is ok